components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/impl/ApplicationsApiServiceImpl.java

```diff
@@ -535,6 +564,13 @@ public String applicationsPostGetLastUpdatedTime(ApplicationDTO body, String con
         return null;
     }
 
+    @Override
+    public String applicationsRegenerateConsumersecretPostGetLastUpdatedTime(ApplicationKeyReGenerateRequestDTO body,
+                                                                             String contentType) {
+
```

Comment:
Could you please explain why it is returning null.? Please add a todo and comment explaining it if it is not yet implemented.